src/App.tsx

```diff
@@ -15,48 +18,108 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  ALPHABETICALLY = 'Sort alphabetically',
+  BY_LENGTH = 'Sort by length',
+  NONE = '',
+}
+
+const sortGoods = (goods: Goods,
+  field: string,
+  isReversed: boolean): Goods => {
+  const sortedGoods = [...goods];
+
+  switch (field) {
+    case SortType.ALPHABETICALLY:
+      sortedGoods.sort((a, b) => a.localeCompare(b));
+      break;
+    case SortType.BY_LENGTH:
+      sortedGoods.sort((a, b) => a.length - b.length);
+      break;
+    default:
+      break;
```

Comment:
don't forget about default case
```suggestion
    default:
      return 0;
```